test/sizes.gzip.json

```diff
@@ -0,0 +1,43 @@
+{
+  "amp.js": "92.33 kB",
```

Comment:
What do you mean "keep it around"? I think the idea is to keep the sizes.txt as is and parse it as a pipe-delimited table. That way we get it to do what we want and keep the history so far as well.